site/gatsby-site/content/about/index.es.mdx

```diff
@@ -26,21 +26,21 @@ Lo invitamos a [explorar los incidentes recopilados hasta la fecha](/apps/discov
 
 La base de datos es un [producto de datos](/develop) y [una colección de aplicaciones](/apps) [en constante evolución](/research/2-roadmap).
```

Comment:
I think where it has `/develop`, it should be `research/2-roadmap`.

Reply:
That link is actually broken on the original page too, let's track that here: https://github.com/responsible-ai-collaborative/aiid/issues/890